docs/src/practices/tracking_issues.md

```diff
@@ -0,0 +1,55 @@
+# Tracking issues
+
+`nearcore` uses tracking issues on github to make it easier to follow progress
+on individual projects.
+
+## Purpose
+
+Different types of audiences want to track progress on feature development on `nearcore`:
+- Other engineers contributing to the feature development but who may have
+  missed some progress while they were unavailable.
+- Pagoda management who would like to have an overview of which teams are
+  working on what features and how much work is left to be done.
+- The general NEAR community who are interested in development of new features
+  in the protocol.
+
+Using tracking issues can help the above audience come up to speed with projects
+quickly and makes it easy for engineers working on projects to share updates.
```

Comment:
I'd reword this a bit, to not be pagoda-specific, and to be more actionable. Something like this:


---

We use so called "tracking issues" to coordinate larger pieces of work (eg, implementation of a new NEP)
Such issues are tagged with `C-tracking-issue`(link to filter) label. 

You can use tracking issues to:

* get a feeling of what's happening in nearcore (by looking at the set of open tracking issues)
* find larger efforts to contribute to (tracking issues usually contain up-for-grabs todo lists)
* follow a progress of a specific feature (you can "subscribe" to an issue on GitHub)

If you are leading or participating in a larger effort, please create a tracking issue for your work.